tests/test_analytics.py

```diff
@@ -0,0 +1,7 @@
+import mapbox
+import pytest
+
+def test_profile_invalid():
+    """'jetpack' is not a valid profile."""
+    with pytest.raises(ValueError):
```

Comment:
Let's add an `InvalidResourceType` error to `mapbox/errors.py` derived from `ValueError` like the existing `InvalidProfileError`.